src/index.js

```diff
@@ -0,0 +1,118 @@
+import { runModal } from './modules/modal.js';
+
+import './style.css';
+import { commentsCounter } from './modules/commentsCounter.js';
+import { likesCounter } from './modules/likesCounter.js';
+import { moviesCounter } from './modules/moviesCounter.js';
+
+const APP_ID = 'DUanzoHMk8l8HLimHh6p';
+let shows;
+
+const fetchAPI = async () => {
+  const baseURL = 'https://api.tvmaze.com/shows';
+  const response = await fetch(baseURL);
+  const data = await response.json();
+
+  shows = data.slice(0, 15);
+  // Fetch likes count for each item and combine with base API data
+  const likes = await getLikes();
+  shows.forEach((show) => {
+    const itemId = show.id;
+    const likeData = likes.find((item) => item.item_id === itemId);
+    show.likes = likeData ? likeData.likes : 0;
+  });
+};
+
+const getLikes = async () => {
+  const res = await fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${APP_ID}/likes`
+  );
+  const data = await res.json();
+  return data;
+};
+
+const updateLikeCount = async (itemId) => {
+  const myHeaders = new Headers();
+  myHeaders.append('Content-Type', 'application/json');
+  myHeaders.append('Authorization', 'Basic bWFqZDowMDAw');
+
+  const raw = JSON.stringify({
+    item_id: Number(itemId),
+  });
+
+  const requestOptions = {
+    method: 'POST',
+    headers: myHeaders,
+    body: raw,
+    redirect: 'follow',
+  };
+
+  fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${APP_ID}/likes`,
+    requestOptions
+  )
+    .then((response) => response.text())
+    .then((result) => console.log(result))
```

Comment:
[REQUIRED] Kindly remove all `console.log` from your code in order to make your project more professional